Formula/libplist.rb

```diff
@@ -1,9 +1,10 @@
 class Libplist < Formula
   desc "Library for Apple Binary- and XML-Property Lists"
   homepage "https://www.libimobiledevice.org/"
-  url "https://github.com/libimobiledevice/libplist/archive/2.2.0.tar.gz"
-  sha256 "7e654bdd5d8b96f03240227ed09057377f06ebad08e1c37d0cfa2abe6ba0cee2"
-  license "LGPL-2.1"
+  url "https://github.com/libimobiledevice/libplist.git",
+      tag:      "2.3.0",
+      revision: "72480212cd8ec0ab8d0b5064df52c146cac603b4"
```

Comment:
Doing a Git checkout just to generate the version is a bit too much. Let's keep the tarball and just set
```ruby
ENV["RELEASE_VERSION"] = version.to_s
```
in the `install` method before calling `autogen.sh` instead.